src/components/form/fields/DateField.tsx

```diff
@@ -29,13 +31,18 @@ export function DateField({
         error: false,
     })
 
-    const { input } = useField<string | undefined>(name, {
-        validate: () => {
-            if (validation.error) {
-                return validation.validationText
-            }
-        },
-    })
+    const { input, meta } = useField<string | undefined>(name)
+
+    useEffect(() => {
```

Comment:
I don't really love this, but I guess with the validation API for the input, this is the easiest way... Doesn't handle change get called when it's cleared?

Reply:
I know i don't either. I think the plan was to add this to the component, since other validations are handled there as well, so was hoping for it to be a temp thing.

Reply:
@Birkbjo let me know if you think what i just pushed is better

Reply:
Yeah, I think thats a bit cleaner - nice! Just some minor lint issues now :)